lib/setup.js

```diff
@@ -36,6 +36,11 @@ var constructResponse = function (server) {
           path: route.path,
           plugin: route.public.realm.plugin || null
         };
+      }).sort(function (a, b) {
```

Comment:
I don't think this is write. For example, the routes `GET /` and `GET /get_address` would be sorted as `GET /`, `GET /get_address`. But `POST /` and `GET /get_address` would be the opposite. I think the correct way would be to `localeCompare()` the path. If the result is 0, then `localeCompare()` the method. Also, `sort()` should return a negative number, 0, or positive number, not a boolean.